arangod/Aql/QueryCursor.cpp

```diff
@@ -128,6 +128,13 @@ Result QueryResultCursor::dumpSync(VPackBuilder& builder) {
 
     builder.add("cached", VPackValue(_cached));
 
+    if (isRetriable()) {
+      _currentBatchResult.first = ++_currentBatchId;
+      if (hasNext()) {
+        builder.add("nextBatchId", _currentBatchId + 1);
```

Comment:
Cursor ids are strings, but the value of `nextBatchId` is a number. Shall we change the value of `nextBatchId` in the response to be a string, too? That would make it consistent. 
I also think this wouldn't cause any problem, because the client application/driver is not supposed to do any math (e.g. +1, -1) with the nextBatchId, but can only resend the very same batch id to the server.